### PR TITLE
lock data.time to a dictionary with t0 and t1

### PR DESCRIPTION
**Pull request by perolavsvendsen**
Following discussions and with reference to #195 this PR updates the metadata examples and the schema to assume data.time as a dictionary.

### CODE REVIEW

File: schema/definitions/0.8.0/examples/surface_seismic_amplitude.yml [hunk 1]
```diff
@@ -138,10 +138,12 @@ data: # The data block describes the actual data (e.g. surface). Only present in
     zmin: 1244.039
     zmax: 2302.683
   time:
```
Comment (perolavsvendsen) [inline schema/definitions/0.8.0/examples/surface_seismic_amplitude.yml:140]: Keeping the `time` block in only one of the examples

**Comment by perolavsvendsen** [comment]
This now becomes a 2-in-1 PR. The actual change is in the metadata examples, schema, and schema tests. Additionally, a set of other files have minor updates making them compliant with the latest version of `black`.

**Review by jcrivenaes — APPROVED** [review]
:+1:

File: schema/definitions/0.8.0/schema/fmu_results.json [hunk 2]
```diff
@@ -80,27 +80,24 @@
         },
         "fmu_time": {
             "title": "FMU time object",
-            "description": "List of time stamps referring to simulated time",
+            "description": "Time stamp for data object.",
             "type": [
                 "object",
-                "array",
                 "null"
```
[the remark this answers is not in the document]
Comment (perolavsvendsen) [reply schema/definitions/0.8.0/schema/fmu_results.json:86]: Not sure if the `null` is needed, no. And I see we have many of those in the rest of the schema as well. I will look into that in a new PR.

**Pull request merged**